README.md

```diff
@@ -0,0 +1,143 @@
+<a name="readme-top"></a>
+
+<div align="center">
+  <h3><b>README Template</b> <h3>
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+- [👥 Authors](#authors)
+- [🤝 Contributing](#contributing)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
```

Comment:
- [ ] Please do not delete any section of the microverse template (FAQ section is optional) as I see you did that, but rather, kindly edit all the sections of your readme and add information relating to your project so that your repo will look more professional 👍🏽. Your readme is missing the future feature section (Add at least two features you will potentially add to your project in the future 😄 ). Kindly add it 👍🏽

Reply:
Hello I'll be sure to set those changes you've proposed to me, I just wanted to ask you if you know a way that I can un merge my branch and add the new changes proposed. Or if you recomend I opean a new branch?